tests/schemas/valid/test_new_calculated_summary_with_dynamic_answers.json

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
We could remove the blocks we don't care about for the tests, to reduce the size of this schema, are there any blocks here that are irrelevant? Same with the other schema.

Reply:
I'm not sure which blocks I can remove without making it an unrealistic schema, I need the add/edit/remove and dynamic answers to have the right setup for the calculated summary, are there any you spotted in particular that I don't need?

Reply:
@katie-gardner I probably saw that list collector driving questions in both schemas, they are not needed, since we only validate unrelated changes.

Reply:
@petechd - sure, sounds good, removed